weldx/welding/groove/iso_9692_1.py

```diff
@@ -240,19 +240,22 @@ def to_profile(self, width_default: pint.Quantity = Q_(5, "mm")) -> geo.Profile:
              pint.Quantity (Default value = Q_(5, "mm"))
 
         """
-        t = self.t.to(_DEFAULT_LEN_UNIT).magnitude
-        b = self.b.to(_DEFAULT_LEN_UNIT).magnitude
-        width = width_default.to(_DEFAULT_LEN_UNIT).magnitude
+        t = self.t
+        b = self.b
+        width = width_default
```

Comment:
I think we can shorten this to one line
```suggestion
        t, b, width  = self.t, self.b, width_default
```

Same for the other assignments